<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="1.1m"
		durationms="63542"
		type="root"
		rid="4f91bb892a35292"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="astan"
		rxversion="8.0.0.0"
		host="CLJSDEASTAN01"
		osversion="Windows 10 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="en-US"
		dtnlanguage="ro-RO"
		screenresolution="3840x1080"
		timestamp="26.01.2018 15:35:44"
		timestampiso="2018-01-26T13:35:44"
		endtime="26.01.2018 15:36:47"
		timeoutfactor="1"
		module="CreateProject">
		<detail>
			CreateProject
		</detail>
		<varbindings>
		</varbindings>
		<item
			timeRelativeToTestSuiteStartTime="01:03.542"
			timeRelativeToTestModuleStartTime="01:03.542"
			timeWallClock="15:36:47"
			time="01:03.542"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
